src/layouts/index.tsx

```diff
@@ -109,50 +114,43 @@ const Layout = ({ children }) => (
             const index = LINKSPATH.findIndex(path => (
               path.path === location.pathname
             ));
+
+            const linkNext = LINKSPATH[index + 1];
+            const linkPrev = LINKSPATH[index - 1];
+            const isPageFirst = index === 0;
+            const isPageLast = index === LINKSPATH.length - 1;
             let labelLeft, labelRight, urlLeft, urlRight;
+            let labelLeftMobile = isPageFirst ? "" : "PREV";
+            let labelRightMobile = (
+              isPageFirst ? "START" : isPageLast ? "HOME" : "NEXT"
+            );
 
-            if (index === -1) return;
-            if (index === 0) {
-              labelLeft = ""
+            if (index === -1) return null;
+            if (isPageFirst) {
```

Comment:
I think `isPageFirst`/`isPageLast` is a bit confusing naming ("hardest problem in computing" and all that).
 `firstIndexActive`/`finalIndexActive`, or maybe `isOnFirstPage`/`isOnFinalPage` is more readable.

Reply:
Yeah, I wasn't super happy with those names but better ones eluded me. I like your 2nd suggestion best.

I generally use a least specific to more specific naming convention (kinda like [BEM](http://getbem.com/naming/)) that I'm still hashing out. Hence `labelLeft`, `labelRight`, `labelLeftMobile`, `labelRightMobile`, and so on. But am not dogmatic.

So would `isOnPageFirst`/`isOnPageFinal` be ok? Alternatively, `indexActiveFirst`/`indexActiveFinal`? And is this a good convention to continue with on this project? I personally find it more readable but don't know how it works for other developers.

Reply:
Yep! Thanks.